lib/Twig/Extension/Core.php

```diff
@@ -1272,7 +1276,11 @@ function twig_length_filter(Twig_Environment $env, $thing)
             return mb_strlen((string) $thing, $env->getCharset());
         }
 
-        return count($thing);
+        if ($thing instanceof \Countable || is_array($thing)) {
+            return count($thing);
+        }
+
+        return 1;
```

Comment:
no, because this would be a BC break (2.x will trigger a deprecation warning there)